### ISSUE TITLE
Answer count 0

### ISSUE THREAD

**Issue by joukyuu** [SubhamTyagi/loco-answers#281]
Most questions answers appear "count 0" in all options...even in easy Google questions...please look into it...

**Comment by pankajksu** [comment]
I agree with u. Same problem mere sath v ho Raha hai. Plssssssss  subham sir fixed this issue.
